baxter_description/launch/rviz_simulation.launch

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+
+<launch>
+  <!-- send robot XML to param server -->
+  <param name="robot_description" command="$(find xacro)/xacro --inorder $(find
```

Comment:
So, I would prefer to approach this delicately, as one can crash Baxter's internal `robot_state_publisher` if they accidentally run this against the real robot. I have addressed this similarly with `sawyer_description`:
https://github.com/RethinkRobotics/sawyer_robot/blob/master/sawyer_description/launch/test_sawyer_description.launch.test
Just like that file, would you mind renaming this launch file to something that `roslaunch` would not find by default, and add a similar warning to the top of the file?